lib/editor-settings.php

```diff
@@ -28,6 +28,9 @@ function gutenberg_extend_post_editor_settings( $settings ) {
 		$settings['defaultTemplatePartAreas'] = gutenberg_get_allowed_template_part_areas();
 	}
 
+	// Some blocks have to use absolute url.
+	$settings['wpAbsoluteAdminUrl'] = admin_url();
```

Comment:
Nitpick: I think maybe we should just name it `adminUrl` to simplify it a bit.

Reply:
I agree. The name you proposed is better.
Fixed in b30ec6bc9acb149caf9ded0d8f615e714eb2964d